src/main.js

```diff
@@ -8,6 +9,12 @@ import App from '@/App.vue';
 const pinia = createPinia();
 
 const app = createApp(App);
+
+// Add rollbar to vue
+if (import.meta.env.VITE_ROLLBAR_ACCESS_TOKEN) {
+  app.use(rollbar);
```

Comment:
I think we can if you want, they might have their own log to say it's enabled. Either way I don't think it's required since it doesn't matter one way or the other.